trio/_core/__init__.py

```diff
@@ -4,6 +4,9 @@
 are publicly available in either trio, trio.lowlevel, or trio.testing.
 """
 
+import typing as _t
+import sys
```

Comment:
I guess technically this should be `import sys as _sys`, though in practice it doesn't do much harm here...

Reply:
Unfortunately, importing `sys` as `_sys` breaks mypy...

Reply:
`del sys` at the end will clean up sufficiently.

Yeah, mypy's platform guards are rather brittle. It's strange since, for example, the magic `from typing import TYPE_CHECKING` seems to work under any name...

Reply:
Thanks, added `del sys`